src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,26 @@
 package practice;
 
-public class CandidateValidator {
+import java.util.function.Predicate;
+import model.Candidate;
 
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE = 35;
+    private static final int REQUIRED_PERIOD = 10;
+    private static final String REQUIRED_NATIONALITY = "Ukrainian";
+
+    @Override
+    public boolean test(Candidate candidate) {
+        return candidate.getAge() >= MIN_AGE
+                && candidate.getNationality().equals(REQUIRED_NATIONALITY)
+                && findPeriodStayInUkr(candidate) >= 10
```

Comment:
don't forget to ue your constants)
```suggestion
                && findPeriodStayInUkr(candidate) >= REQUIRED_PERIOD
```